### PR TITLE
Sync .idea/compiler.xml.

### PR DESCRIPTION
**Pull request by motlin**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -7,12 +7,12 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="eclipse-collections-api" />
+        <module name="jcstress-tests" />
```
Comment (donraab) [inline .idea/compiler.xml:10]: The modules are the same, just in a different order. Does order matter here?

Comment (motlin) [reply .idea/compiler.xml:10]: Order doesn't matter. I was seeing churn in this file. Does it happen for you?

Comment (donraab) [reply .idea/compiler.xml:10]: How do we know this change won't cause churn for someone else? Do you know why the order is being changed on your machine?

Comment (motlin) [reply .idea/compiler.xml:10]: I don't know. I assume it's related to the version of IntelliJ. I'd like to know what you see on your machine. Does this file churn? Does this commit remove the diffs and show your file as unchanged again?

Comment (donraab) [reply .idea/compiler.xml:10]: Yes, it churns, but differently than what you are committing.

Comment (mohrezaei) [reply .idea/compiler.xml:10]: It's weird to see you going back and forth on something that's been obvious for 2 decades...
https://github.com/eclipse/eclipse-collections/issues/581

Comment (motlin) [reply .idea/compiler.xml]: I deleted compiler.xml and the Inspections builds still pass. So I committed that and .gitignored it.

**Review by donraab — APPROVED** [review]
LGTM

**Pull request merged**